catalog/ui/search-ui/standard/src/main/webapp/js/model/Query.js

```diff
@@ -70,6 +71,7 @@ define([
                 this.listenTo(this, 'change:usng', this.setRadiusUsng);
                 this.listenTo(this, 'EndExtent', this.notDrawing);
                 this.listenTo(this, 'BeginExtent', this.drawingOn);
+                this.listenTo(wreqr.vent, 'query:clearFilters', this.clearFilters);
```

Comment:
Tse 'search:clearfilters' instead of 'query:clearFilters'. The first part of the event is the module that this event is coming from, which is the search module and the second part should just be some non-camelcased string.